example.cfg

```diff
@@ -0,0 +1,23 @@
+{
+	"http": "localhost:5566",
+	"workdir": "/home/jeremy/go/src/github.com/google/syzkaller/workdir",
+	"sshkey": "/home/jeremy/.ssh/id_rsa",
+	"vmlinux": "/home/jeremy/vm/sles12-sp2-beta1-fuzz/vmlinux",
+	"image": "/home/jeremy/vm/sles12-sp2-beta1.qcow2",
+	"initrd": "/home/jeremy/vm/sles12-sp2-beta1-fuzz/initrd-4.4.13-68-default",
```

Comment:
I am not sure it's worth adding to the example. All users managed to do without it to date. Please remove.